src/renderer/components/SidePanel/ScreensPanel.vue

```diff
@@ -60,7 +60,8 @@ export default {
       this.$store.dispatch("artboards/addArtboard", {
         title: "Untitled",
         width: 375,
-        height: 667
+        height: 667,
+        isVisible: true
```

Comment:
I think this makes sense given the current implementation, no changes needed. 

FYI in the future, I would like to improve the way Vuex State and (what should maybe be a Class) is handled, because it could be very easy to forget to add/remove one of these properties if the data model changes.